### PR TITLE
chore: test serving audio

### PR DESCRIPTION
**Pull request by tomasciccola**
add a test and an audio test file

### CODE REVIEW

File: tests/fixtures/blob-api/audio.mp3
Comment (EvanHahn) [inline tests/fixtures/blob-api/audio.mp3:1]: incredible choice. 10/10

**Review by EvanHahn — APPROVED** [review]
LGTM.

It seems like [GitHub Actions are having issues](https://www.githubstatus.com/)...maybe we can wait to see if those resolve before merging?

File: tests/fixtures/blob-api/audio.mp3
Comment (tomasciccola) [reply tests/fixtures/blob-api/audio.mp3:1]: heh, its the whilhelm scream, a mytical audio sample that appears in A LOT of movies

**Comment by tomasciccola** [comment]
> It seems like [GitHub Actions are having issues](https://www.githubstatus.com/)...maybe we can wait to see if those resolve before merging?

Mhm, it seems that the fastify test is hanging and not related with the CI issues. Its hanging on the test after the one I've added (`retrieving style.json using stable url`). 
I wonder whats that about...

**Comment by tomasciccola** [comment]
I was forgetting to stop the fastify controller..

**Review by EvanHahn — APPROVED** [review]

**Pull request merged**